python/ray/data/_internal/plan.py

```diff
@@ -399,14 +408,24 @@ class OneToOneStage(Stage):
     def __init__(
         self,
         name: str,
-        block_fn: Callable[[Block], Block],
-        compute: str,
+        block_fn: BlockTransform,
+        compute: Union[str, ComputeStrategy],
         ray_remote_args: dict,
+        fn: Optional[UDF] = None,
+        fn_args: Optional[List[Any]] = None,
+        fn_kwargs: Optional[Dict[str, Any]] = None,
+        fn_constructor_args: Optional[List[Any]] = None,
+        fn_constructor_kwargs: Optional[Dict[str, Any]] = None,
     ):
         super().__init__(name, None)
         self.block_fn = block_fn
         self.compute = compute or "tasks"
         self.ray_remote_args = ray_remote_args or {}
+        self.fn = fn
+        self.fn_args = fn_args
+        self.fn_kwargs = fn_kwargs
+        self.fn_constructor_args = fn_constructor_args
+        self.fn_constructor_kwargs = fn_constructor_kwargs
 
     def can_fuse(self, prev: Stage):
```

Comment:
Can you add some tests for fusion case? The current tests have just one mapping operation, add some more to invoke this code.